### ISSUE TITLE
Investigate strategies to fully implement POSIX [.collating-element.] syntax.

### ISSUE THREAD

**Issue by mikehaertel** [mikehaertel/minrx#14]

**Comment by arnoldrobbins** [comment]
@bhaible A GLIBC API is necessary but not sufficient (as my high school math teacher used to say).  We need a POSIX API in order to be able to work portably on macOS and legacy Unixes (and maybe even Windows).

**Comment by bhaible** [comment]
> We need a POSIX API

Right. But the Austin Group rarely standardizes an API that no libc has implemented so far. Therefore the most promising approach for getting a POSIX API is to get a glibc API first, and then (with the help of Eric Blake) push that onto the Austin Group's table.

**Comment by arnoldrobbins** [comment]
I see. That makes sense.  Thanks.  Same note applies in #15 .
